src/components/Input/index.js

```diff
@@ -82,19 +96,22 @@ class Input extends Component {
 Input.defaultProps = {
   disabled: false,
   htmlProps: {},
-  error: false,
   errorMessage: '',
 }
 
 Input.propTypes = {
+  id: PropTypes.string,
+  value: PropTypes.string,
+  placeholder: PropTypes.string,
+  type: PropTypes.string,
+  step: PropTypes.string,
```

Comment:
Add comments to Proptypes to help understand their meaning. Ex: https://github.com/vtex/styleguide/blob/feature/tooltip/src/components/ToolTip/index.js#L56